rotimi-best/classroomio
# feat: add dropdown to community ask

## What does this PR do?



Fixes #31 



## Type of change




## How should this be tested?



- Go to the community/ask route, select a drop down, ask the question and click on publish then go to community to see the questions but this time there's a small link to course of the respective question.

## Checklist



### Required


### Appreciated

![Screenshot (552)](https://github.com/rotimi-best/classroomio/assets/88969825/f6e0c806-69e8-4039-bf23-972b4a15b1b1)
